packages/app-sdk/src/utils/localization.ts

```diff
@@ -71,9 +71,15 @@ export function getLocalizedImageByType(
   }
 
   const imagesByCorrectOrientation = allImages.filter(i => i.orientation === orientation.toUpperCase());
-  const imagesByCorrectType = imagesByCorrectOrientation.filter(i => i.type === imageType);
-  if (imagesByCorrectType.length > 0) {
-    return imagesByCorrectType[0];
+  const imageByCorrectType = imagesByCorrectOrientation.find(i => i.type === imageType);
+  if (imageByCorrectType) {
+    return imageByCorrectType;
+  }
+  if (imageType !== "cover") {
```

Comment:
It makes no sense to me to hardcode "cover" here. Can we either add a new argument for `fallbackImageType?: string` or maybe `falbackImageTypes?: string[]`

Reply:
I've added imageTypeFallback as a string because I don't think there is a need to add several fallbacks to imageType, but if you think this is a needed feature I can still add it.

Reply:
I agree, we probably don't need more than one 👍